docs/S3Installation.md

```diff
@@ -0,0 +1,20 @@
+# S3 Integration - Installing Boto
+
+## Introduction ##
+Boto is a Python library that is required in order for DIGITS to interact with S3. This is not required if DIGITS is being trained on local files but is required for retrieving files from any S3 endpoint.
+
+## Installation ##
```

Comment:
Could you add an example walk-through that shows how to populate an S3 bucket with the expected contents and then how to load the data in DIGITS? It would be useful for laymen like me who are not so familiar with all of this :-)

Reply:
We added some instructions to examples/s3/README.md and an accompanying link in the base README.md. Please let us know if any of it could use further detail.